Add release notes for 5.14.2 (#1897)
sensu/sensu-docs: content/sensu-go/5.15/reference/entities.md

## Usage limits

> Sensu's free entity limit is 100 entities. All [license-activated features][9] are available for free in the packaged Sensu Go distribution up to an entity limit of 100. If your Sensu instance includes more than 100 entities, [contact us][10] to learn how to upgrade your installation and increase your limit. See [the announcement on our blog][11] for more information about our usage policy.

Author: ```suggestion
This version of Sensu has no functional limitations based on entity count. If your Sensu instance includes over 1,000 entities, contact us to learn about [license-activated features][9] designed for monitoring at scale. See [Discourse][10] for more information about our usage policy. 
```

[api-filter]: ../../api/overview#filtering
[sensuctl-filter]: ../../sensuctl/reference#filtering

> [9]: ../../getting-started/enterprise

Author: ```suggestion
[9]: ../../getting-started/enterprise
[10]: https://discourse.sensu.io/t/introducing-usage-limits-in-the-sensu-go-free-tier/1156
[11]: ../checks#round-robin-checks
[12]: ../../guides/monitor-external-resources/#using-a-proxy-entity-to-monitor-a-website
[13]: ../checks/#proxy-requests
[14]: ../../guides/monitor-external-resources/
[15]: #proxy-entities
```

[api-filter]: ../../api/overview#filtering
[sensuctl-filter]: ../../sensuctl/reference#filtering
[9]: ../../getting-started/enterprise

> [11]: https://blog.sensu.io/one-year-of-sensu-go

Author: ```suggestion
```

[api-filter]: ../../api/overview#filtering
[sensuctl-filter]: ../../sensuctl/reference#filtering
[9]: ../../getting-started/enterprise

> [10]: https://sensu.io/contact

Author: ```suggestion
```

[13]: ../checks/#proxy-requests
[14]: ../../guides/monitor-external-resources/
[15]: #proxy-entities

> 

Author: ```suggestion
```
